src/fondant/pipeline.py

```diff
@@ -75,6 +79,38 @@ def __init__(
         self.p_volumes = p_volumes
         self.ephemeral_storage_size = ephemeral_storage_size
 
+    def _set_arguments(
+        self,
+        arguments: t.Optional[t.Dict[str, t.Any]],
+    ) -> t.Dict[str, t.Any]:
+        """Set component arguments based on provided arguments and relevant ComponentOp
+        parameters.
+        """
+
+        def _validate_partition_size_arg(file_size):
```

Comment:
Can we add this as a `type` when registering it on the parser in the `Executor` class?